block/manager.go

```diff
@@ -194,6 +194,7 @@ func (m *Manager) Start(ctx context.Context) error {
 	/* ----------------------------- sequencer mode ----------------------------- */
 	// Subscribe to batch events, to update last submitted height in case batch confirmation was lost. This could happen if the sequencer crash/restarted just after submitting a batch to the settlement and by the time we query the last batch, this batch wasn't accepted yet.
 	go uevent.MustSubscribe(ctx, m.Pubsub, "updateSubmittedHeightLoop", settlement.EventQueryNewSettlementBatchAccepted, m.UpdateLastSubmittedHeight, m.logger)
+	go uevent.MustSubscribe(ctx, m.Pubsub, "newBondedSequencer", settlement.EventQueryNewBondedSequencer, m.UpdateSequencerSet, m.logger)
```

Comment:
what about full node mode? probably required there as well

Reply:
true, moved to before to be called by sequencers and full-nodes